core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,599 @@
+| `fieldName`  | `string`            | ✅        | Name of the field to edit                         |
+| `mode`       | `'plain' \| 'full'` | ❌        | Editor mode (default: 'plain')                    |
+| `format`     | `'text' \| 'html'`  | ❌        | Output format (default: 'text')                   |
+
+**Usage**:
 
-`src/app/app.config.ts`
 ```typescript
-import { InjectionToken } from '@angular/core';
-import { ClientConfig, DotCmsClient } from '@dotcms/client';
+import { DotCMSBasicContentlet } from '@dotcms/types';
+import { DotCMSEditableText } from '@dotcms/angular/next';
+
+@Component({
+    selector: 'app-your-component',
+    imports: [DotCMSShowWhen],
+    template: `
+        <dotcms-editable-text
+            [contentlet]="contentlet"
+            fieldName="title"
+            mode="plain"
+        />
+    `
+})
+export class YourComponent {
+    @Input() contentlet: DotCMSBasicContentlet;
+}
+```
 
-export const DOTCMS_CLIENT_TOKEN = new InjectionToken<DotCmsClient>('DOTCMS_CLIENT');
+**Editor Integration**:
 
-export const appConfig: ApplicationConfig = {
-    providers: [
-        provideRouter(routes),
-        {
-            provide: DOTCMS_CLIENT_TOKEN,
-            useValue: DotCmsClient.init(DOTCMS_CLIENT_CONFIG),
-        }
-    ],
+* Detects UVE edit mode and enables inline TinyMCE editing
+* Saves on blur without needing full content dialog
+
+**Common Issues**:
+
+* `mode` and `format` must match the actual field type
+* `full` mode is only supported for `WYSIWYG` fields. For `text` and `text_area` fields, use `plain` mode.
+* Should not be used on Block Editor fields
+
+
+#### DotCMSBlockEditorRenderer
+
+**Overview**: Component for rendering Block Editor content from dotCMS with support for custom block renderers.
+
+**Inputs**:
+
+| Input              | Type                  | Required | Description                                      |
+|-------------------|----------------------|-----------|--------------------------------------------------|
+| `blocks`          | `BlockEditorContent`  | ✅        | The block editor content to render               |
+| `customRenderers` | `CustomRenderers`     | ❌        | Custom rendering functions for specific blocks   |
+| `className`       | `string`             | ❌        | CSS class to apply to the container              |
+| `style`           | `CSSProperties`       | ❌        | Inline styles for the container                  |
+
+**Usage**:
+
+```typescript
+import { DotCMSBasicContentlet } from '@dotcms/types';
+import { DotCMSBlockEditorRenderer } from '@dotcms/angular/next';
+
+const myCustomRenderers = {
+    customBlock: import('./custom-block.component').then(c => c.CustomBlockComponent)
 };
+
+@Component({
+    selector: 'app-your-component',
+    imports: [DotCMSShowWhen],
+    template: `
+        <dotcms-block-editor-renderer
+            [blocks]="contentlet.myBlockEditorField"
+            [customRenderers]="myCustomRenderers"
```

Comment:
How this looks like?

Reply:
https://github.com/dotCMS/core/blob/1d16ce19955aef5b4d21ce8027055b706ca48867/core-web/libs/sdk/angular/README.md?plain=1#L289-L291